bedrock/firefox/templates/firefox/australis/tour-base.html

```diff
@@ -23,8 +23,6 @@
 
 {% block body_class %}sky{% endblock %}
 
-{% block site_css %}{% endblock %}
```

Comment:
So the `firefox/australis/firstrun-tour.html` page has broken/missing CSS, but I think only because it's using the `site_css` block. Testing locally, switching `firefox/australis/firstrun-tour.html` to use `page_css` instead fixes the issue.